stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewHighlighter.java

```diff
@@ -102,15 +105,48 @@ private void highlightViewOnUiThread() {
       }
 
       if (mHighlightedView != null) {
-        mHighlightedView.getOverlay().remove(mHighlightDrawable);
+        for (ColorDrawable drawable : mHighlightDrawables) {
+          mHighlightedView.getOverlay().remove(drawable);
+        }
       }
 
-      if (viewToHighlight != null) {
-        mHighlightDrawable.setColor(mContentColor.get());
-        mHighlightDrawable.setBounds(0, 0, viewToHighlight.getWidth(), viewToHighlight.getHeight());
-        viewToHighlight.getOverlay().add(mHighlightDrawable);
+      if (viewToHighlight != null &&
+          viewToHighlight.getLayoutParams() instanceof ViewGroup.MarginLayoutParams) {
```

Comment:
This doesn't seem right to me.  What if this view is in a context where its parent doesn't have `MarginLayoutParams`?  It is still highlightable, no, just perhaps not with this new margin highlighting?